subsys/net/lib/sockets/sockets_inet.c

```diff
@@ -2584,6 +2653,23 @@ int zsock_setsockopt_ctx(struct net_context *ctx, int level, int optname,
 
 			break;
 
+		case IP_RECVTTL:
+			if (IS_ENABLED(CONFIG_NET_IPV4) &&
+				IS_ENABLED(NET_CONTEXT_RECV_HOPLIMIT)) {
+				ret = net_context_set_option(ctx,
+								 NET_OPT_RECV_HOPLIMIT,
+								 optval,
+								 optlen);
```

Comment:
Alignment might be wrong here (could be also that the web page just shows this incorrectly)

Reply:
There is definitely something fishy with the alignment. The code should look like this with proper indentation
```
	case IP_RECVTTL:
		if (IS_ENABLED(CONFIG_NET_IPV4) &&
		    IS_ENABLED(CONFIG_NET_CONTEXT_RECV_HOPLIMIT)) {
			ret = net_context_set_option(ctx,
						     NET_OPT_RECV_HOPLIMIT,
						     optval,
						     optlen);
```
It looked like there are too many tabs used, have you set tab size to 8 in your editor.